lib/easypost/beta/referral.rb

```diff
@@ -107,4 +107,52 @@ def self.add_credit_card(referral_api_key, number, expiration_month, expiration_
     response = create_easypost_credit_card(referral_api_key, stripe_credit_card_token, priority)
     EasyPost::Util.convert_to_easypost_object(response, referral_api_key)
   end
+
+  # Add a Stripe payment method to a Referral Customer. This function requires the Referral Customer's API key.
+  # @param [String] stripe_customer_id Unique customer ID provided by Stripe.
+  # @param [String] payment_method_reference ID of the card or bank account provided by Stripe.
+  # @param [String] payment_method_type Which priority to save this payment method as on EasyPost, either 'primary' or 'secondary'.
+  # @param [String] api_key Override the API key used for this request.
+  # @return [EasyPost::PaymentMethod] The newly-added payment method.
+  # noinspection RubyParameterNamingConvention
+  def self.add_payment_method(stripe_customer_id, payment_method_reference, priority = 'primary', api_key = nil)
+    wrapped_params = {
+      payment_method: {
+        stripe_customer_id: stripe_customer_id,
+        payment_method_reference: payment_method_reference,
+        priority: priority.downcase,
+      },
+    }
+    response = EasyPost.make_request(:post, '/beta/referral_customers/payment_method', api_key, wrapped_params)
+    # noinspection RubyMismatchedReturnType
+    EasyPost::Util.convert_to_easypost_object(response, api_key)
+  end
+
+  # Refund a Referral Customer's wallet by a specified amount. Refund will be issued to the user's original payment method.
+  # This function requires the Referral Customer's API key.
+  # @param [Integer] amount The amount to refund, in cents.
+  # @param [String] api_key Override the API key used for this request.
+  # @return [EasyPost::Beta::PaymentRefund] The newly-created refund.
+  def self.refund_by_amount(amount, api_key = nil)
+    params = {
+      refund_amount: amount,
+    }
+    response = EasyPost.make_request(:post, '/beta/referral_customers/refunds', api_key, params)
+    # noinspection RubyMismatchedReturnType
+    EasyPost::Util.convert_to_easypost_object(response, api_key) # TODO: Needs "object" or ID prefix to determine object class.
```

Comment:
re: TODO -- There is no EasyPost object that is associated with this response. These functions in our lib fallback to returning arrays/json if it can't convert to an object so we shouldn't need to define one because we don't have one.

Reply:
See https://github.com/EasyPost/easypost-ruby/pull/236